Set libdirs and bindirs as empty in header only packages (#3179)
conan-io/docs: tutorial/creating_packages/other_types_of_packages/header_only_packages.rst

            # This will also copy the "include" folder
            copy(self, "*.h", self.source_folder, self.package_folder)

        def package_info(self):
            # For header-only packages, libdirs and bindirs are not used

>             # so it's recommended to set those as empty.

Reviewer: ```suggestion
            # so it's necessary to set those as empty.
```

            self.cpp_info.bindirs = []
            self.cpp_info.libdirs = []

> Please, note that we are setting cpp_info.bindirs and cpp_info.libdirs to ``[]`` because
> we won't use them for header-only.

Reviewer: ```suggestion
Please, note that we are setting ``cpp_info.bindirs`` and ``cpp_info.libdirs`` to ``[]`` because
header-only libraries don't have compiled libraries or binaries, but they default to ``["bin"]``, and ``["lib"]``, then it is necessary to change it.
```

            # This will also copy the "include" folder
            copy(self, "*.h", self.source_folder, self.package_folder)

        def package_info(self):
            # For header-only packages, libdirs and bindirs are not used

>             # so it's recommended to set those as empty.

Reviewer: ```suggestion
            # so it's necessary to set those as empty.
```
